### ISSUE TITLE
Add the video thumbnail as `attachement` in AP representation of a video

### ISSUE THREAD

**Issue by Aldarone** [Chocobozzz/PeerTube#453]
Mastodon uses the `attachment` property of the AP object that represents a toot to display embedded media files.

```
"attachment": [
        {
            "mediaType": "image/png",
            "name": "Long press on a tab to open the menu",
            "type": "Document",
            "url": "https://files.mastodon.social/media_attachments/files/003/332/746/original/d311fa1b93733f6d.png"
        },
]
```

Currently, the thumbnail is in the `icon` property of the AP representation of a video : 

```
    "icon": {
        "height": 110,
        "mediaType": "image/jpeg",
        "type": "Image",
        "url": "https://peertube.mastodon.host/static/thumbnails/c5eb4428-4412-47ba-b509-85aea25d3a08.jpg",
        "width": 200
    },
```

If Peertube adds the `attachment` property to embed the video thumbnail(s?) it would enhance the experience of the federation.

**Comment by Gargron** [comment]
I disagree for multiple reasons. "icon" for thumbnail is way more semantic than just "attachment". Secondly, it doesn't even matter because Mastodon has special handling for Video objects to convert them to a text status internally, so you could fix this on our end anyway. And lastly, a toot that has an image attached will not display a link card preview, which in case of PeerTube is the embedded player.
